rubocop/rubocop.yml

```diff
@@ -12,6 +12,10 @@ Layout/FirstArrayElementIndentation:
 Layout/MultilineMethodCallIndentation:
   EnforcedStyle: indented
 
+Lint/UselessAssignment:
+  Exclude:
+    - "spec/**/*"
```

Comment:
I appreciate at least this being scoped to the spec directory only (if this gets merged, could extend to test for minitest).